compiler/src/iree/compiler/Codegen/Common/GPU/AMDGPUDistributeContract.cpp

```diff
@@ -6,7 +6,7 @@
 
 #include "iree/compiler/Codegen/Common/GPU/GPUVectorDistribution.h"
 #include "iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.h"
-#include "iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUInterfaces.h"
+#include "iree/compiler/Codegen/Utils/VectorOpUtils.h"
```

Comment:
Strange that this file is called `AMDGPUDistributeContract.cpp` and is `Common/GPU`. Not nevessary here, but probably needs to be changed.

Reply:
Yeah I was trying to put the pass in `LLVMGPU` but it's quite tricky due to the messy transform dialect related dependency graphs. I wasn't able to make it work so gave up. Also commented here: https://github.com/openxla/iree/pull/16445#discussion_r1493075502